cunumeric/coverage.py

```diff
@@ -157,7 +162,7 @@ def unimplemented(
 
     if reporting:
 
-        @wraps(func)
+        @wraps(func, assigned=_UNIMPLEMENTED_COPIED_ATTRS)
```

Comment:
I haven't used `assigned` before. Reading the docs, it seems like this will skip copying `__doc__` in all cases. Is that actually what is desired? (Or am I misunderstanding what `assigned` does?)

Reply:
I think you have it right, although I'm just doing this for unimplemented functions, that we pull from the cloned module. I don't personally think it's a big loss, because the NumPy docstrings are generally not 100% applicable to cuNumeric (e.g. they talk about limitations that apply to NumPy and not cuNumeric, or they document options that we don't support). We could maybe just replace those with a stub "This function is unimplemented, and will fall back to <link-to-corresponding-numpy-function>". Or we could just filter out the functions that we know have bad docstrings (running the risk of needing to update this filter as the docstrings of underlying functions change). What do you think?

Reply:
Removing them all is fine I just wanted to make sure that was the intent 👍

Reply:
It was actually not too hard to add a stub docstring. It looks good per-function, but weirdly I don't see summary sentences on the function list. Can you take another look?

<img width="882" alt="Screenshot 2023-08-11 at 4 07 37 PM" src="https://github.com/nv-legate/cunumeric/assets/215728/a991e3bf-9d0e-4197-ae14-21f4156623b0">

<img width="882" alt="Screenshot 2023-08-11 at 4 07 45 PM" src="https://github.com/nv-legate/cunumeric/assets/215728/0f21dd46-776c-4c6a-8297-ca5777246e2d">